### PR TITLE
Add warnings about validating the IAM role in the Cloud9 IDE

### PR DESCRIPTION
**Pull request by buzzsurfr**
*Issue #, if available:*

*Description of changes:*
Add warnings about validating the IAM role in the Cloud9 IDE

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by geremyCohen** [comment]
@buzzsurfr you may want to add some language around if the sts call doesn't return the instance-id based arn immediately, to wait, as it may take some time for that button event to propagate

File: content/prerequisites/workspaceiam.md [hunk 1]
```diff
@@ -28,3 +28,31 @@ echo "export AWS_REGION=${AWS_REGION}" >> ~/.bash_profile
...
+
+#### INVALID
+
+If the _Arn contains `TeamRole` or `MasterRole`, <span style="color: red;">**DO NOT PROCEED**</span>. Go back and confirm the steps on this page.
+
+```output
+{
+    "Account": "123456789012", 
+    "UserId": "AROA1SAMPLEAWSIAMROLE:i-01234567890abcdef", 
+    "Arn": "arn:aws:sts::123456789012:assumed-role/TeamRole/MasterRole"
```
Comment (dalbhanj) [inline content/prerequisites/workspaceiam.md]: I think the output here would differ if user is going through the workshop on their own. We should either clarify in the instructions or stay neutral and just ask to validate by providing an alternative way to get the same outcome (see below)

File: content/prerequisites/workspaceiam.md [hunk 2]
```diff
@@ -28,3 +28,31 @@ echo "export AWS_REGION=${AWS_REGION}" >> ~/.bash_profile
...
+
+#### VALID
+
+If the _Arn_ contains `modernizer-workshop-cl9` and an Instance ID, you may proceed.
+
+```output
+{
+    "Account": "123456789012", 
+    "UserId": "AROA1SAMPLEAWSIAMROLE:i-01234567890abcdef", 
+    "Arn": "arn:aws:sts::123456789012:assumed-role/modernizer-workshop-cl9/i-01234567890abcdef"
```
Comment (dalbhanj) [inline content/prerequisites/workspaceiam.md]: Can we provide alternative way to get correct info by providing steps similar to this link: https://eksworkshop.com/pr-195/logging/prereqs/
This way the user can validate the role attached to Cloud9 is the same as get-caller-identity call

File: content/eksctl/launcheks.md [hunk 3]
```diff
@@ -4,6 +4,25 @@ date: 2018-08-07T13:34:24-07:00
 weight: 20
 ---
 
+
+{{% notice warning %}}
+**DO NOT PROCEED** with this step unless you have [validated the IAM role](/prerequisites/workspaceiam/#validate-the-iam-role) in use by the Cloud9 IDE. You will not be able to run the necessary kubectl commands in the later modules unless the EKS cluster is built using the IAM role.
+{{% /notice %}}
+
+{{%expand "How do I check the IAM role on the workspace?" %}}
```
Comment (dalbhanj) [inline content/eksctl/launcheks.md]: You should use the Challenge template specified here: https://github.com/aws-samples/eks-workshop/wiki/Workshop-testing

File: content/prerequisites/workspaceiam.md (hunk 1 above)
Comment (buzzsurfr) [reply content/prerequisites/workspaceiam.md]: Done in future commit.

File: content/eksctl/launcheks.md (hunk 3 above)
Comment (buzzsurfr) [reply content/eksctl/launcheks.md]: I'm concerned using a challenge question here. This is a critical step that should not be missed.

**Pull request merged**